test/CMakeLists.txt

```diff
@@ -15,6 +15,18 @@
 set (ranlux_24_48_test.pass_timeout_debug "900") # 15min
 set (ranlux_24_48_test.pass_timeout_release "720") # 12min
 
+# If TBB headers are available, libstdc++ enables TBB backend; some TBB symbols become undefined unless it is linked.
+# The undefined symbol error arises only with GCC compiler.
+if (CMAKE_CXX_COMPILER_ID STREQUAL GNU AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 9)
+    if (ONEDPL_BACKEND MATCHES "^(serial|omp|dpcpp_only)$")
+        find_package(TBB)
```

Comment:
Do we want to add `QUIET` here (especially with the status message below)?

Also, is the `tbbmalloc`  component required to resolve this linker issue or just `tbb`? It seems like if no components are listed, `FindTBB.cmake` requires `tbb` and `tbbmalloc` by default.  If only `tbb` is required, then we could perhaps add `REQUIRED tbb` to this line as well.

Reply:
> Do we want to add QUIET here (especially with the status message below)?

Indeed, let's add QUIT.

> Also, is the `tbbmalloc` component required to resolve this linker issue or just `tbb`?

I am not sure if `libstdc++` uses `tbbmalloc`. It will be safer to link against it the way it is linked for TBB backend in oneDPL.

>  If only tbb is required, then we could perhaps add REQUIRED tbb to this line as well.

Adding `REQUIRED` will result in a critical error if oneTBB is not found. It should be optional with the current logic: if tbb is found linked it, if it not - do not link it.